### ISSUE TITLE
`React.Fragment` is a valid return type, even though it is not a type declared by React

### ISSUE THREAD

**Issue by TomasBarry** [SublimeLinter/SublimeLinter-flow#73]
The following code is deemed ok by _SublimeLinter-flow_:

```es6
// @flow

import * as React from 'react'

const X = (): React.Fragment => (
  <React.Fragment>
    <p>
      { 'Hello :)' }
    </p>
  </React.Fragment>
)

export default X

```

However, [`React.Fragment` is a function, not a type](ad608421aa5f6a7754037227d806999d17b12bf5). As such, _SublimeLinter-flow_ should warn against using `React.Fragment` as a return type of `X`.

This is further highlighted by using the same code snippet on [Flow's online test editor](https://flow.org/try/#0PTAEAEDMBsHsHcBQiCWBbADrATgF1AFSgCGAzqAEoCmxAxvpNrGqAOTY32vK2wB2pfAA1QAXlAAKAJQAuSp1wA6AGLZiAczRU++UQD5JiUKAA81OktUatOvUeOmMdhw4DebABJVocUDKmsoAC+9sYmwE724eb0Kmqa2rh2UshUAB5YeKAAJlSQxACu0MKIQA).

**Comment by kaste** [comment]
Can you make a short repro repo on GitHub?

**Comment by TomasBarry** [comment]
@kaste, [here](https://github.com/TomasBarry/SublimeLinter-flow-issues-73) is a repo you can clone and load in Sublime Text. You'll notice that neither ESLint nor Flow raises any flags even though there should be one for the return type.

Any thoughts on this?

**Comment by kaste** [comment]
You know what?

![image](https://user-images.githubusercontent.com/8558/61486107-fd3de980-a9a2-11e9-8a7e-ad3fb2d0fb27.png)

💩 

I used the following flowconfig

```
[ignore]

[include]

[libs]

[lints]

[options]

[strict]
```

Debug output

```
SublimeLinter: #68 linter.py:1578     Running ...

  D:\SublimeLinter-flow-issues-73  (working dir)
  > type index.js | D:\SublimeLinter-flow-issues-73\node_modules\.bin\flow.CMD check-contents D:\SublimeLinter-flow-issues-73\index.js --show-all-errors --json

SublimeLinter: #68 linter.py:1155     flow: output:
  [{"flowVersion":"0.102.0","jsonVersion":"1","errors":[{"kind":"infer","level":"error","suppressions":[],"message":[{"context":"const X = (): React.Fragment => (","descr":"Cannot use function type as a type because function type is a value. To get the type of a value use `typeof`.","type":"Blame","loc":{"source":"D:\\SublimeLinter-flow-issues-73\\index.js","type":"SourceFile","start":{"line":5,"column":15,"offset":56},"end":{"line":5,"column":28,"offset":70}},"path":"D:\\SublimeLinter-flow-issues-73\\index.js","line":5,"endline":5,"start":15,"end":28}]}],"passed":false},{}]
SublimeLinter: flow 1 errors. passed: False 
SublimeLinter: flow line: 4, col: 14, level: error, message: React.Fragment (Cannot use function type as a type because function type is a value. To get the type of a value use `typeof`.) 
```

**Comment by TomasBarry** [comment]
It actually appears to be an issue with older versions of Flow not flagging this as an issue.
